src/containers/pages/FocusInvestigation/active/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -106,18 +95,7 @@ exports[`containers/pages/ActiveFocusInvestigation renders ActiveFocusInvestigat
         "calls": Array [
           Array [
             0,
-            Object {
```

Comment:
I think that this change should not happen.  Have we stopped calling the Superset service with this filter?